danger_code_ws/1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <cstdio>
+
+int main() {
+    char buffer[10];
+
+    std::cout << "Enter a string: ";
+    fgets(buffer, sizeof(buffer), stdin); // Safer alternative to gets
```

Comment:
This line exceeds 80 characters.  Consider breaking it up for better readability.  For example, you could put `stdin` on a new line.